app/src/interfaces/list-o2m/list-o2m.vue

```diff
@@ -1,515 +1,335 @@
 <template>
```

Comment:
Seems like `totalCount` is `"100"` for a list-o2m with a single item, not sure why yet

Reply:
I know: `existingItemCount.value` is a string (`1`), and the logic is `"1" + 0 + 0` = `"100"`

Reply:
Haha, love those javascript pearls. 😄